plugins/org.eclipse.n4js.flowgraphs/src/org/eclipse/n4js/flowgraphs/analysers/UsedBeforeDeclaredAnalyser.java

```diff
@@ -24,12 +24,13 @@
 import org.eclipse.n4js.n4JS.VariableDeclaration;
 
 /**
- * This graph visitor prints all paths
+ *
  */
-public class CheckVariableGraphVisitor extends GraphVisitor {
+public class UsedBeforeDeclaredAnalyser extends GraphVisitor {
```

Comment:
Is this analyzer only for variables or also for constants?

Reply:
True. For all where an error marker should be shown, I think.